lib/parser.js

```diff
@@ -1,41 +1,86 @@
 'use strict';
 
+const PROTOCOL_VERSION = 1;
```

Comment:
In issue #3 2b for protocol version declared

Reply:
We will use `writeInt16LE` to write this `Number` value to 2 bytes binary field @mille-nium